app/src/main/res/layout/mentor_recycleview.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you rename this file to something like `mentor_list_item` so we know that it's for an item, not the recycler view itself?